### ISSUE TITLE
Feature request: Include JavaScript REPL wrapper for `jsc`

### ISSUE THREAD

**Issue by lxgr** [holzschu/a-shell#129]
Given the inclusion of `jsc`, it would seem possible to include a simple REPL wrapper to allow running JavaScript expressions on the command line.

**Comment by holzschu** [comment]
Good idea. I'll have a look.
It might be difficult to run these in a separate JS context, so they would be sharing variables between windows and instances of the REPL.

**Comment by lxgr** [comment]
Oh, interesting, so is the similarity between the `jsc` command on a Mac and in a-Shell in name only? As in, your version is just a hook to basically execute a provided .js file in a preexisting, global JavaScriptCore instance, rather than fork/exec'ing a new process of `jsc`? I just had a look at `ios_system`, and it seems like fork isn't even possible, so that makes more sense to me now.

I think even a REPL with this caveat of shared global variables might be useful!

**Comment by holzschu** [comment]
Yes, the whole project started because there is no fork/exec on iOS...

**Comment by holzschu** [comment]
Hi, this issue was fixed by #195, #207, #211 and #218. It is now available in the TestFlight version: https://testflight.apple.com/join/WUdKe3f4

**Comment by lxgr** [comment]
What's the command that it's available under? Is it still in the current TestFlight build? At a first glance I couldn't find it.

**Comment by holzschu** [comment]
`jsi` will start a JavaScript REPL interpreter.
